modules/gazelle/0.40.0/patches/module_dot_bazel_version.patch

```diff
@@ -0,0 +1,14 @@
+===================================================================
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -4,9 +4,9 @@
+     version = "",
+     repo_name = "bazel_gazelle",
+ )
+ 
+-bazel_dep(name = "bazel_features", version = "1.9.1")
++bazel_dep(name = "bazel_features", version = "0.40.0")
```

Comment:
@kormide Looks like the version regex doesn't like `version` to be set to an empty string explicitly. I am going to work around this in https://github.com/bazel-contrib/bazel-gazelle/pull/1974.